examples/vue/src/App.vue

```diff
@@ -16,7 +16,7 @@ export default {
 </script>
 
 <style>
-@import "../node_modules/@elastic/react-search-ui-views/lib/styles/styles.css";
+@import "@elastic/react-search-ui-views/lib/styles/styles.css";
```

Comment:
This import didn't work before when I was running the demo from the repo root, now both ways of running work.